### ISSUE TITLE
Showing  java.lang.OutOfMemoryError: Java heap space in windows

### ISSUE THREAD

**Issue by iamashiq** [epirus-io/epirus-cli#53]
do i miss any settings or configuration ?

Exception in thread "main" java.lang.OutOfMemoryError: Java heap space
        at org.bouncycastle.crypto.generators.SCrypt.SMix(Unknown Source)
        at org.bouncycastle.crypto.generators.SCrypt.MFcrypt(Unknown Source)
        at org.bouncycastle.crypto.generators.SCrypt.generate(Unknown Source)
        at org.web3j.crypto.Wallet.generateDerivedScryptKey(Wallet.java:145)
        at org.web3j.crypto.Wallet.create(Wallet.java:80)
        at org.web3j.crypto.Wallet.createStandard(Wallet.java:98)
        at org.web3j.crypto.WalletUtils.generateWalletFile(WalletUtils.java:81)
        at org.web3j.crypto.WalletUtils.generateNewWalletFile(WalletUtils.java:72)
        at org.web3j.crypto.WalletUtils.generateFullNewWalletFile(WalletUtils.java:50)
        at org.web3j.crypto.WalletUtils.generateNewWalletFile(WalletUtils.java:63)
        at io.epirus.console.project.wallet.ProjectWallet.<init>(ProjectWallet.java:41)
        at io.epirus.console.account.AccountUtils.accountDefaultWalletInit(AccountUtils.java:47)
        at io.epirus.console.project.InteractiveOptions.createWallet(InteractiveOptions.java:104)
        at io.epirus.console.EpirusCommand.maybeCreateDefaultWallet(EpirusCommand.java:127)
        at io.epirus.console.EpirusCommand.parse(EpirusCommand.java:110)
        at io.epirus.console.Epirus.main(Epirus.java:17)

**Comment by josh-richardson** [comment]
This is likely due to a 32-bit JVM. Please ensure that you run Epirus using a 64-bit JVM.

**Comment by iamashiq** [comment]
Can you please help me on how to configure this. I installed by running the
powershell command in github.

**Comment by josh-richardson** [comment]
You need to make sure that the JVM referenced by your PATH variable in Windows points to a 64-bit JVM.

**Comment by iikirilov** [comment]
@iamashiq did you manage to sort out this issue?

**Comment by iamashiq** [comment]
I was in a hurry that day because next day I had to submit my project. So I
used my old web3j cli. So I didn't sort out this on my system.

For JVM, I'm using 64-bit windows and Android studio running fine. Hope the
JVM is 64-bit. And i tried increasing allocated memory for jvm.

I tried epirus to get ABIv2 compiled, so that day i installed epirus
successfully on a Mac Pc. But it didn't  work and shown same incompatiblity
towards ABIv2 like old web3j.

So I rewrite my solidity contract without ABIv2. Isn't it compatible with
epirus ?
